src/SFA.DAS.Reservations.Data.UnitTests/AccountLegalEntityRepository/WhenStoringAccountLegalEntities.cs

```diff
@@ -51,22 +48,20 @@ public async Task Then_The_AccountLegalEntity_Is_Added_To_The_Repository()
                 AgreementSigned = true,
                 AccountLegalEntityName = "Test"
             };
+            
+            _dataContext.Setup(x => x.AccountLegalEntities)
+                .ReturnsDbSet(new List<AccountLegalEntity>());
+            _dataContext.Setup(x => x.Database)
+                .Returns(_dataFacade.Object);
+            _accountLegalEntityRepository = new Repository.AccountLegalEntityRepository(_dataContext.Object);
 
             //Act
             await _accountLegalEntityRepository.Add(expectedAccountLegalEntity);
 
             //Assert 
-            _dataContext.Verify(x => x.AccountLegalEntities.AddAsync(It.Is<AccountLegalEntity>(
-                c => c.Id.Equals(expectedAccountLegalEntity.Id) &&
-                     c.AccountId.Equals(expectedAccountLegalEntity.AccountId) &&
-                     c.AgreementSigned.Equals(expectedAccountLegalEntity.AgreementSigned) &&
-                     c.AccountLegalEntityId.Equals(expectedAccountLegalEntity.AccountLegalEntityId) &&
-                     c.AccountLegalEntityName.Equals(expectedAccountLegalEntity.AccountLegalEntityName) &&
-                     c.LegalEntityId.Equals(expectedAccountLegalEntity.LegalEntityId) &&
-                     c.ReservationLimit.Equals(expectedAccountLegalEntity.ReservationLimit) 
-                     ), It.IsAny<CancellationToken>()), Times.Once);
```

Comment:
what's your thinking behind removing this verify? It's still being called in the implementation.